### ISSUE TITLE
Wrong probabilities for xgboost classifier

### ISSUE THREAD

**Issue by solegalli** [TeamHG-Memex/eli5#255]
Hello. 
I am trying eli on the titanic dataset. I can see that eli explains fairly well when the probability of survival is high or low (although it did not coincide exactly with the output of my xgboost model). 

But when the probability of survival is around 0.5, it makes very biased predictions. Is Eli supported for XGBClassification?

Also, the output of Eli.explain_prediction changes depending on whether the probability of survival is low or high. If p(0) is high, Eli will output weights to predict p(0), but if the P(1) is high, then it will output weights for p(1)? This makes it difficult to incorporate Eli in an automated process.
Thank you
Sole

**Comment by solegalli** [comment]
Sorted. Never mind. Sorry.

**Comment by noleto** [comment]
it may help others in the future => #274

@solegalli I suspect what you described comes from the fact that the score displayed by `eli5.explain_prediction` does not take into account for XGBoost `base_score`. So it does not match exactly the output of your XGBoost model). 
I was experiencing the same issue until I understood I have to sum `base_score` to the score given by ELI5 (as shown here: https://github.com/dmlc/xgboost/issues/1274)
